x-pack/plugins/task_manager/server/task.ts

```diff
@@ -334,12 +343,27 @@ export interface ConcreteTaskInstance extends TaskInstance {
   /**
    * The state passed into the task's run function, and returned by the previous
    * run. If there was no previous run, or if the previous run did not return
-   * any state, this will be the empy object: {}
+   * unknown state, this will be the empy object: {}
    */
+  // we allow any here as unknown will break current use in otehr plugins
+  // this can be fixed by supporting generics in the future
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
   state: Record<string, any>;
 
   /**
    * The random uuid of the Kibana instance which claimed ownership of the task last
    */
   ownerId: string | null;
 }
+
+export type SerializedConcreteTaskInstance = Omit<
+  ConcreteTaskInstance,
+  'state' | 'params' | 'scheduledAt' | 'startedAt' | 'retryAt' | 'runAt'
+> & {
+  state: string;
+  params: string;
+  scheduledAt: string;
+  startedAt: string | null;
+  retryAt: string | null;
+  runAt: string;
+};
```

Comment:
removing `any` revealed that the typing in our serialisation was wrong - this adds a missing type that correctly types the serialised shape.